catalog/spatial/geocoding/spatial-geocoding-plugin/src/main/java/org/codice/ddf/spatial/geocoding/plugin/GeoCoderPlugin.java

```diff
@@ -63,9 +61,9 @@ public CreateRequest process(CreateRequest input)
             return input;
         }
 
-        geoCoder = geoCoderFactory.getService();
+        GeoCoder geoCoder = geoCoderFactory.getService();
```

Comment:
This is a typo, the geoCoder should have been passed into the setCountryCode method.